### ISSUE TITLE
Screen reader does not read top prompt in Profiler Settings window

### ISSUE THREAD

**Issue by eclectir** [Microsoft/PTVS#2257]
Prompt highlighted below

![image](https://cloud.githubusercontent.com/assets/5177491/23766752/e9b9413c-04ba-11e7-9608-f09be2333c28.png)

**Comment by huguesv** [comment]
You need to use CAPS LOCK+LEFT ARROW or CAPS LOCK + RIGHT ARROW to make it read the text.
